Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.2.4-slim-bookworm
+FROM ruby:3.0-slim-bullseye
```

Comment:
Also curious what prompted this change!

Reply:
i forgot to back this out when reverted the ruby update! so it was incompatible with our gemfile/gemfile.lock dep versions